core/services/ocr2/plugins/ocr2vrf/juelsfeecoin/link_eth_price_provider_test.go

```diff
@@ -38,4 +38,26 @@ func Test_JuelsPerFeeCoin(t *testing.T) {
 		assert.Nil(t, price)
 		assert.Equal(t, "get aggregator latest answer: network failure", err.Error())
 	})
+
+	t.Run("returns juels per fee coin", func(t *testing.T) {
+		latestRoundData := aggregator_v3_interface.LatestRoundData{Answer: big.NewInt(10000)}
+		mockAggregator.On("LatestRoundData", mock.Anything).Return(latestRoundData, nil)
+		p := linkEthPriceProvider{aggregator: mockAggregator, stubbed: true}
+		price, err := p.JuelsPerFeeCoin()
+
+		require.NoError(t, err)
+		assert.Equal(t, int64(0), price.Int64())
+		mockAggregator.AssertNotCalled(t, "LatestRoundData")
+	})
+
+	t.Run("returns error when contract call fails", func(t *testing.T) {
```

Comment:
same question as above. maybe one of the test cases is not needed?

Reply:
Yes if you see I assert `mockAggregator.AssertNotCalled(t, "LatestRoundData")` while still running the normal flow of the non-stubbed version. This is to ensure that the stubbed version is not going through with the RPC call while all other things are identical.